system/system.pb.go

```diff
@@ -254,12 +255,14 @@ var (
 		1: "SIGNAL_TERM",
 		2: "SIGNAL_KILL",
 		3: "SIGNAL_HUP",
+                4: "SIGNAL_ABRT",
 	}
 	KillProcessRequest_Signal_value = map[string]int32{
 		"SIGNAL_UNSPECIFIED": 0,
 		"SIGNAL_TERM":        1,
 		"SIGNAL_KILL":        2,
 		"SIGNAL_HUP":         3,
+                "SIGNAL_ABRT":        4,
```

Comment:
tabs instead of spaces
```suggestion
		4: "SIGNAL_ABRT",
	}
	KillProcessRequest_Signal_value = map[string]int32{
		"SIGNAL_UNSPECIFIED": 0,
		"SIGNAL_TERM":        1,
		"SIGNAL_KILL":        2,
		"SIGNAL_HUP":         3,
		"SIGNAL_ABRT":        4,
```

Reply:
Fixed